PopPUNK/network.py

```diff
@@ -399,33 +469,74 @@ def constructNetwork(rlist, qlist, assignments, within_label,
                                                                                 weights = True)
             for (ref, query, weight) in zip(extra_sources, extra_targets, extra_weights):
                 edge_tuple = (ref, query, weight)
-                connections.append(edge_tuple)
+                if ref < query:
```

Comment:
This stops edges being added twice I guess? Is it possible to have the input to this be query > ref though, and what should happen then?

Reply:
Was keeping consistent with code above, but that proved to be erroneous, because sparse matrices are usually not symmetrical. Removed conditions in 55507b4.